packages/components/src/modelLoader.ts

```diff
@@ -0,0 +1,45 @@
+import { INodeOptionsValue } from './Interface'
+import axios from 'axios'
+
+/* This is a temp URL, will be replaced with the actual URL from the main repo */
+const MASTER_MODEL_LIST =
+    'https://raw.githubusercontent.com/vinodkiran/Flowise/FEATURE/externalize-model-list/packages/components/models.json'
```

Comment:
reminder to change this path

Reply:
Review and merge https://github.com/FlowiseAI/Flowise/pull/2153. Post that we can update this path to the updated config file.

Reply:
PR merged. we can update the path now